### ISSUE TITLE
Dl error on this video code cb5LZbhatD4

### ISSUE THREAD

**Issue by n00dlefr34k** [ytdl-org/youtube-dl#31935]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
G:\tunez\__ytdl>youtube-dl.exe -x --verbose --audio-format "mp3" https://www.youtube.com/watch?v=cb5LZbhatD4
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-x', '--verbose', '--audio-format', 'mp3', 'https://www.youtube.com/watch?v=cb5LZbhatD4']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.22621
[debug] exe versions: ffmpeg N-94345-g1123331f59, ffprobe N-94345-g1123331f59
[debug] Proxy map: {}
[youtube] cb5LZbhatD4: Downloading webpage
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\youtube.py", line 1794, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 1012, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.


G:\tunez\__ytdl>youtube-dl.exe --version
2021.12.17
```

## Description

This URL doesn't work for me.  Here is the entire command for you to review.
youtube-dl.exe -x --verbose --audio-format "mp3" https://www.youtube.com/watch?v=cb5LZbhatD4

Best of luck

**Comment by dirkf** [comment]
>[x] I've searched the bugtracker for similar bug reports including closed ones

Read #30839 (**again**?) to find out how.

**Comment by n00dlefr34k** [comment]
You are right.  I did some other search and got thishttps://github.com/ytdl-org/youtube-dl/issues/31530sorry for the duplicatepleases close  From: dirkfSent: Saturday, March 25, 2023 6:05 PMTo: ytdl-org/youtube-dlCc: n00dlefr34k; AuthorSubject: Re: [ytdl-org/youtube-dl] Dl error on this video code cb5LZbhatD4 (Issue #31935) [x] I've searched the bugtracker for similar bug reports including closed onesRead #30839 (again?) to find out how.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
